### PR TITLE
more log targets, include: cert-manager, letsencrypt CLusterIssuer

### PR DESCRIPTION
**Pull request by rob2universe**

### CODE REVIEW

**Comment by rob2universe** [comment]
@upgradingdave please review

File: include/cert-manager.mk [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	helm repo update
+	helm install cert-manager jetstack/cert-manager \
+	--namespace cert-manager \
+  	--create-namespace \
+  	--version v1.9.1 \
+	--set installCRDs=true
+
+.PHONY: letsencypt-staging
+letsencypt-staging:
+	cat ../include/letsencypt.yaml | sed -E "s/someone@somewhere.io/$(certEmail)/g" | kubectl create -n cert-manager -f -
```
Comment (falko) [inline include/cert-manager.mk:16]: Should `someone@somewhere.io` be a configuration variable defined by the user in the `Makefile`?

File: include/ingress-nginx.mk [hunk 2]
```diff
@@ -3,11 +3,10 @@ ingress: ingress-nginx camunda-values-nginx.yaml
 
 .PHONY: ingress-nginx
 ingress-nginx:
-	-kubectl create namespace ingress-nginx
 	helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
 	helm repo update ingress-nginx
 	helm search repo ingress-nginx
-	helm install ingress-nginx ingress-nginx/ingress-nginx --namespace ingress-nginx
+	helm install ingress-nginx ingress-nginx/ingress-nginx --namespace ingress-nginx --create-namespace --wait
```
Comment (falko) [inline include/ingress-nginx.mk:9]: Great! Let's do this everywhere.

**Review by falko — APPROVED** [review]
Looks good to me.

File: include/cert-manager.mk (hunk 1 above)
Comment (rob2universe) [reply include/cert-manager.mk:16]: This is the placeholder string being replaced with the variable certEmail from the make file

**Review by upgradingdave — APPROVED** [review]

**Comment by upgradingdave** [comment]
Merged these changes into PR #36

**Pull request closed without merging**